### PR TITLE
[WIP] Adding new procedures to the Eclipse doc

### PR DESCRIPTION
**Pull request by mpershina**
Tracked under [MTA-3533](https://issues.redhat.com/browse/MTA-3533)

The review can also be performed in the google doc. New procedures:

- [2.1. Installing Java Development Kit](https://docs.google.com/document/d/18L1eilLAz7wCzfjWOCC-WDASYuPRAzvSdS-oxXV_onE/edit#heading=h.5qyz93sz658i)
- [2.2. Installing Eclipse](https://docs.google.com/document/d/18L1eilLAz7wCzfjWOCC-WDASYuPRAzvSdS-oxXV_onE/edit#heading=h.nffldu6cems4)
- [2.3 Adding Jboss Tools dependencies for the MTA plug-in](https://docs.google.com/document/d/18L1eilLAz7wCzfjWOCC-WDASYuPRAzvSdS-oxXV_onE/edit#heading=h.v1y950pgl2wl)

Updated procedure (includes the Important note): 

- [2.4. Installing in a connected environment](https://docs.google.com/document/d/18L1eilLAz7wCzfjWOCC-WDASYuPRAzvSdS-oxXV_onE/edit#heading=h.ldadwsqkrerr)

### CODE REVIEW

File: docs/topics/eclipse-installing-plugin.adoc [hunk 1]
```diff
@@ -9,14 +9,16 @@
...
+The {PluginName} has been tested with the Eclipse integrated development environment (IDE) for Java Enterprise Developers 2023-03.
 
 .Prerequisites
 
 include::snippet_jdk-hardware-mac-prerequisites.adoc[leveloffset=0]
 
 * link:https://www.eclipse.org/downloads/packages/release/2023-03/r/eclipse-ide-java-developers[Eclipse IDE for Java Enterprise Developers 2023-03]
 * JBoss Tools, installed from the link:https://marketplace.eclipse.org/content/jboss-tools[Eclipse Marketplace]
++
+IMPORTANT: You must also add JBoss Tools dependencies that are required by Eclipse during the {ProductShortName} plug-in installation process. For more information, see xref:adding-jboss-tools-dependencies-for-the-mta-plug-in_eclipse-code-ready-studio-guide[Adding Jboss Tools dependencies for the MTA plug-in].
```
Comment (anarnold97) [inline docs/topics/eclipse-installing-plugin.adoc:21]:
```suggestion
[IMPORTANT]
====
You must also add JBoss Tools dependencies that are required by Eclipse during the {ProductShortName} plug-in installation process. For more information, see xref:adding-jboss-tools-dependencies-for-the-mta-plug-in_eclipse-code-ready-studio-guide[Adding JBoss Tools dependencies for the {ProductShortName} plug-in].
====
```

So pedantic, I annoy myself

File: docs/topics/installing-java-development-kit.adoc [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+* Oracle JDK 11 							
+* Oracle JDK 17 							
+* Eclipse Temurin™ JDK 11 							
+* Eclipse Temurin™ JDK 17 		
+
+.Procedure
+
+. From the link:https://www.oracle.com/java/technologies/downloads/#java11[Oracle JDK download page], download the JDK `tar.gz` file suitable for your system.
++ 
+Note that you need to accept the Oracle license agreement to proceed.
```
Comment (anarnold97) [inline docs/topics/installing-java-development-kit.adoc:21]:
```suggestion
[Note]
====
You need to accept the Oracle license agreement to proceed.
====
```
Pedantic, feel free to ignore

File: docs/topics/installing-eclipse.adoc [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+. Optional: To launch Eclipse from you application menu, create a desktop entry:
+
+.. Create a new `eclipse.desktop` file:
++
+[source,terminal,subs="attributes+"]
+----
+sudo nano /usr/share/applications/eclipse.desktop
+----
+
```
Comment (anarnold97) [inline docs/topics/installing-eclipse.adoc:53]:
```suggestion

+
Or use your default terminal text editor, instead of nano

```

**Review by anarnold97 — APPROVED** [review]
LGTM - made a few pedantic comments to show i was paying attention

File: docs/topics/eclipse-installing-plugin.adoc (hunk 1 above)
Comment (mpershina) [reply docs/topics/eclipse-installing-plugin.adoc:21]: Haha, it's actually better for writers if they want to make some fixes since it's more visible :-D

**Review by dymurray — APPROVED** [review]

File: docs/topics/installing-eclipse.adoc [hunk 4]
```diff
@@ -0,0 +1,68 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-08-29
+:_mod-docs-content-type: PROCEDURE
+
+[id="installing-eclipse_{context}"]
+= Installing Eclipse
+
+Eclipse is an integrated development environment (IDE) that you can use to install the {ProductFullName} play-in used for migrating and modernizing applications.
```
Comment (mpershina) [inline docs/topics/installing-eclipse.adoc:8]:
```suggestion
Eclipse is an integrated development environment (IDE) that you can use to install the {ProductFullName} plug-in used for migrating and modernizing applications. 			
```

**Pull request merged**